src/Datadog.Trace.ClrProfiler.Managed/Integrations/SqlServer.cs

```diff
@@ -19,30 +16,33 @@ public static class SqlServer
         /// </summary>
         /// <param name="this">The "this" pointer for the method call.</param>
         /// <param name="behavior">The behavior.</param>
-        /// <param name="method">The method.</param>
         /// <returns>The original methods return.</returns>
-        public static object ExecuteReader(dynamic @this, int behavior, string method)
+        public static object ExecuteReader(dynamic @this, int behavior)
         {
+            var command = (DbCommand)@this;
```

Comment:
can we do this? Doesn't this force us to bind to a specific version of System.Data? Maybe it doesn't matter because its a core library?

Reply:
Before last night, I assumed that in .NET Core, `DbCommand` and `CommandBehavior` required the `System.Data.Common` NuGet package, but apparently they are now included with .NET Core 2.0, otherwise this wouldn't even build for `netcoreapp2.0`. I just looked and found it in `C:\Users\lucas\.nuget\packages\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\System.Data.Common.dll`. That folder seems to contain all the "core" libraries.

For .NET Framework this is fine, since `System.Data.dll` is definitely always included.

About binding to specific versions, .NET uses a type-forwarding system that resolves version mismatches at run time. That's how you can run an application built for older versions of .NET on newer CLRs even when the assembly versions have changed. It's also how you can the `SqlClient` that's built-in in .NET Framework or the ones from NuGet in the same shared library.